### ISSUE TITLE
 IP search and language fails until any number is entered in IP field and saved

### ISSUE THREAD

**Issue by jenskastensson** [nabbl/node-red-contrib-google-home-notify#9]
The IP search feature is a very nice but it doesn't work a 100% in my environment.
These are my steps:
1.install the "google home notify" from palette
2. restart node-red server
3. add a gh-n node to a flow, open config and press search
I see this in the console:
vendor.js:5 GET http://192.168.1.67:1880/languages 404 (Not Found)
vendor.js:5 GET http://192.168.1.67:1880/ipaddresses 404 (Not Found)
The search doesn't show any results, 
4. save and restart, still no search result
5. enter anything ('1' e.g.) in the IP and Save
6. Now the search result is available Also the languages are listed.

Node-RED version: v0.17.5
Node.js  version: v6.11.2
Linux 3.2.40 arm LE

**Comment by nabbl** [comment]
There is a problem where the GET commands are not returning anything the first time they are called.

I will look into this but I can't promise a fix anytime soon.

**Comment by Haringstad** [comment]
Yup, no languages can be chosen. And I am stuck to the stupid UK female voice..... She sounds like she's drunk! Any way of forcing US english?
